isoCluster.py

```diff
@@ -112,7 +106,7 @@ def extract_hostnames_from_file(file_path: str) -> list[str]:
             if line.strip().startswith('host'):
                 match = re.search(r'host\s+(\S+)', line.strip())
                 if match:
-                    hostnames.append(match.group(1))            
+                    hostnames.append(match.group(1))
```

Comment:
could you rebase the part of the patch with mypy/flake8/black fixes to the parent commit that introduces the issues?

Reply:
Yes good point done